email_service_client/client/resources.py

```diff
@@ -21,17 +35,43 @@ def __init__(self, session: requests.Session) -> None:
         super().__init__(session)
         self.resource_url = f"{API_BASE_URL}/email-identities"
 
+
+    @raise_http_error
     def list(self) -> List[EmailIdentity]:
-        return self._session.get(self.resource_url).json()
+        return self._session.get(self.resource_url) #type: ignore
+    
+    @raise_http_error
+    def get(self, id: int) -> EmailIdentity:
+        return self._session.get(f"{self.resource_url}/{id}") #type: ignore
 
-    def get(self, id) -> EmailIdentity:
-        return self._session.get(f"{self.resource_url}/{id}").json()
+    @raise_http_error
+    def create(self, data: CreateEmailIdentityPayload) -> dict:
 
-    def create(self, data: CreateEmailIdentityPayload):
         encoded_data = requests_toolbelt.MultipartEncoder(fields=data)
         self._session.post(
             self.resource_url,
             data=encoded_data,
             headers={"Content-Type": encoded_data.content_type},
         )
-        return self._session.post(self.resource_url).json()
+        return self._session.post(self.resource_url) #type: ignore
+
+    @raise_http_error
+    def configure_sending(self, id: int) -> dict:
+        return self._session.post(f"{self.resource_url}/{id}/configure/sending") #type: ignore
```

Comment:
What's the error causing `#type: ignore` here? It's in every line.

Reply:
The return type is `dict`, but I am returning `Response` here. The decorator will do the `response.json()` part of things.